packages/observable/src/subscribable.ts

```diff
@@ -3,38 +3,40 @@ import {
     arrayRemoveItem, objectForEach, options
 } from '@tko/utils'
 
-import Subscription from './Subscription'
-import { SUBSCRIBABLE_SYM } from './subscribableSymbol'
+import { default as Subscription, LATEST_VALUE } from './Subscription'
 import { applyExtenders } from './extenders.js'
 import * as dependencyDetection from './dependencyDetection.js'
 export { isSubscribable } from './subscribableSymbol'
+import { SUBSCRIBABLE_SYM } from './subscribableSymbol'
 
 // Descendants may have a LATEST_VALUE, which if present
 // causes TC39 subscriptions to emit the latest value when
 // subscribed.
-export const LATEST_VALUE = Symbol('Knockout latest value')
 
-export function subscribable () {
+export function subscribable<T> (this: Subscribable<T>) : void {
   Object.setPrototypeOf(this, ko_subscribable_fn)
   ko_subscribable_fn.init(this)
 }
 
-export var defaultEvent = 'change'
+interface TC39Callback {
+}
+type Subscriber = (value: any) => void | TC39Callback
 
-var ko_subscribable_fn = {
+export const defaultEvent = 'change'
+
+const ko_subscribable_fn = {
```

Comment:
By defining all of the types in an `interface` or `type` (e.g. `ISubscribable<T>`) and then assigning `ko_subscribable_fn` to that (i.e. `ko_subscribable_fn: ISubscribable<T>`), `T` becomes accessible to all of the members below (almost all of the `any`s below are actually `T`). `this` can by typed as well if desired, and `ISubscribable<T>` becomes reusable.

Reply:
I just noticed it's already defined and used in Observable. Just move it here and consolidate and it's good.

Reply:
@caseyWebb When I try
```ts
const ko_subscribable_fn: Subscribable<T> = {
```
I get a `Cannot find name T`.  Staring at it now. :)

Reply:
@brianmhunt see this commit https://github.com/caseyWebb/tko/commit/2096642c83b8310bbd4029b4bc5434fe67e51455